src/js/utils/buffer.js

```diff
@@ -30,13 +28,11 @@ export function bufferedPercent(buffered, duration) {
   }
 
   for (let i = 0; i < buffered.length; i++) {
-    start = buffered.start(i);
-    end = buffered.end(i);
+    const start = buffered.start(i);
+    const end = buffered.end(i);
 
     // buffered end can be bigger than duration by a very small fraction
-    if (end > duration) {
-      end = duration;
-    }
+    end = Math.min(end, duration);
```

Comment:
This tries to redefine a const

Reply:
That's unbelievable mistake, I've fixed it.